example_config

```diff
@@ -7,28 +7,37 @@ path /static StaticHandler {
 	root testFiles1;
 }
 
+path /static1 StaticHandler {
+	root testFiles1/markdown/Readme.md;
+}
+
+path /static2 StaticHandler {
+	root testFiles1/markdown/Readme.md;
```

Comment:
Maybe we can set this to just  `testFiles1/markdown` so we can serve any markdown files we put into that directory?